### DISCUSSION TITLE
XAML Rule

### DISCUSSION THREAD

**Discussion by AlleSchonWeg** [RehanSaeed/EditorConfig#28] [Q&A]
### Question

Hi,
it is possible to declare a rule for xaml? I need rules for Attribute Spacing and Element Spacing.

**Comment by RehanSaeed** [comment]
I have no idea if you can do that in `.editorconfig`. As long as you have a warning code, I'm assuming you can change the log level. Give it a try!

**Comment by RehanSaeed** [comment]
Closing for now. Feel free to reopen if you have anything to add.
